packages/prisma/migrations/20240221100119_add_home_view_field/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "users" ADD COLUMN     "homeView" TEXT NOT NULL DEFAULT 'event-types';
```

Comment:
instead of adding this to the database, what if we just save this setting on localStorage?

seems to do the job and does not pollute the database

Reply:
its not worth to sync this setting across devices

Reply:
Ok let's do with localStorage

Reply:
may be if you want tell me

Reply:
yes please use localStorage (there is a hook for it already) and not change the database 🙏

Reply:
Ok

Reply:
can i use cookies for storing the values ?

Reply:
because i can't able to get localStorage in server side component in index.tsx file that redirect the user
`import type { GetServerSidePropsContext } from "next";

import { getServerSession } from "@calcom/features/auth/lib/getServerSession";

function RedirectPage() {
  return;
}

export async function getServerSideProps({ req, res }: GetServerSidePropsContext) {
  const session = await getServerSession({ req, res });

  if (!session?.user?.id) {
    return { redirect: { permanent: false, destination: "/auth/login" } };
  }

  return { redirect: { permanent: false, destination: "/event-types" } };
}

export default RedirectPage;
`